### ISSUE TITLE
Dutch translation

### ISSUE THREAD

**Issue by KeesWD** [braginteractive/materialwp#31]
Hi,
Like the theme!
Translated into Dutch. How to name .po and .mo files to get the translation activated?
Tried materialwp-nl_NL to no avail. Also just plain nl_NL doesn't work.
Hope to hear.

Cheers,
KeesWD

**Comment by braginteractive** [comment]
Haven't translated the theme. I would recommend doing some Google searches around this topic.

**Comment by KeesWD** [comment]
Hi Brad,

Thanks for your immediate answer.

I translated the included .pot file through poedit.
Question is how to name the .po and .mo file to get translation activated.

Cheers,
KeesWD

2016-10-26 18:10 GMT+02:00 Brad Williams notifications@github.com:

**Comment by braginteractive** [comment]
Yeah, I dunno. I have never done it. I would recommend reading some translation tutorials on the interwebs.

**Comment by KeesWD** [comment]
Okay, thanks so far.
I'll keep you informed.

2016-10-26 18:27 GMT+02:00 Brad Williams notifications@github.com:

**Comment by braginteractive** [comment]
Please do for other looking for the same.

**Comment by KeesWD** [comment]
Things I found so far:
you have a textdomain defined in functions.php, OK
.po and. mo files have to be named [in my situation] nl_NL.xx

What I can't find is the translation for READ MORE [css to capitalize] any
idea where to find this?

2016-10-26 18:44 GMT+02:00 Brad Williams notifications@github.com:
